node/resolvers/catalog/category.ts

```diff
@@ -39,10 +47,7 @@ export const resolvers = {
     slug: async ({ id }: any, _: any, { dataSources: { catalog } }: any) => {
       const categories = await catalog.categories(3) as Category[]
 
-      const flattenCategories = categories.reduce(
-        (acc : Category[], c) => acc.concat(c, c.children),
-        []
-      )
+      const flattenCategories: Category[] = flatten(categories)
```

Comment:
Is this necessary also on line [#65](https://github.com/vtex-apps/store-graphql/pull/285/files#diff-c155ec5068061a2e88950287bf79667cR65)?

Reply:
And also in line 27.